.github/workflows/release-30_publish_release_draft.yml

```
@@ -197,9 +197,13 @@ jobs:
         if: ${{ matrix.target == 'aarch64-apple-darwin' }}
         working-directory: ${{ github.workspace}}/release-artifacts/${{ matrix.target }}/${{ matrix.binary }}
         run: |
+          . ../../../.github/scripts/common/lib.sh
```

This is probably more readable and secure:
```suggestion
          . ${{ github.workspace}}/.github/scripts/common/lib.sh
```